### PR TITLE
Fix/design issues

### PR DESCRIPTION
**Pull request by jacob-rachal**
There are still a few things I want to test/play around with, but for all intents and purposes this thing is now mobile-friendly. (I think I have some redundant code in the CSS that needs trimming.)

### CODE REVIEW

**Review by anthony61016 — APPROVED** [review]
looks good

File: index.html [hunk 1]
```diff
@@ -14,18 +14,19 @@
 </head>
 <body class="background">
 
-	<nav class="text-center">
-	  <ul>
+	<nav class="text-center navbar">
+	  <!-- <ul> -->
```
Comment (rehazler) [inline index.html]: No need to comment stuff out. If you have confirmed that it is unnecessary, then just remove it entirely.

File: css/styles.css [hunk 2]
```diff
@@ -2,13 +2,29 @@ body {
...
+  font-family: 'happy monkey';
+  font-style: bold;
+}
 
+/* styling for the header/menu nav. */
 .topnav {
   font-size: 43px;
   font-family: "happy monkey";
-  padding: 3px 53px;
+  /* padding: 3px 53px; */
```
Comment (rehazler) [inline css/styles.css]: Same as the index file, trash the comment clutter.

**Review by rehazler — CHANGES_REQUESTED** [review]
In general it looks good from what I can tell. Only thing, across the files, there is no need to comment stuff out. If you have confirmed that it is unnecessary, then just remove it entirely. Trash the comment clutter.

**Review by rehazler — APPROVED** [review]
Lookin' good!

**Pull request merged**